src/modules/main/sidebar/Sidebar.scss

```diff
@@ -26,14 +26,7 @@
           border-radius: 0;
           color: $primary;
 
-          & > span {
```

Comment:
Undo these changes.
Additionally, we also need the labels you removed

Reply:
Reverted changes